.github/workflows/push-release.yml

```diff
@@ -0,0 +1,31 @@
+name: any-pr
```

Comment:
Same comment -> "Publish stable release to NPM"

Reply:
Commented above